### ISSUE TITLE
Treesitter does not handle [private] alias

### ISSUE THREAD

**Issue by AddictArts** [IndianBoy42/tree-sitter-just#158]
It seems the treesitter will error on a private alias. Using InspectTree I see the error, remove the private anno and fixed.
Example:

```
alias foo := bar
[private]
alias o := open
```

**Comment by tgross35** [comment]
What version and editor are you using? This works correctly when I try with the latest `main` and Helix 

<img width="199" alt="image" src="https://github.com/IndianBoy42/tree-sitter-just/assets/13724985/fdc644a4-574a-4eb8-8ae9-e927895c3778">

**Comment by tgross35** [comment]
A screenshot would be helpful too

**Comment by tgross35** [comment]
@AddictArts in case you didn't see the above

**Comment by AddictArts** [comment]
> @AddictArts in case you didn't see the above

Thank you. I will get a better example, it will take some time though. Hopefully on Monday I can. Don't have the machine available to get it. But we are doing some fairly advanced justfile stuff, so I wasn't surprised. Apologies that I made an assumption on the issue.

**Comment by AddictArts** [comment]
@tgross35 Found the issue, I couldn't share the file, updated the issue to the below

It seems the treesitter will error on a private alias. Using InspectTree I see the error, remove the private anno and fixed.
Example:

```
alias foo := bar
[private]
alias o := open
```

**Comment by kennyp** [comment]
This seems to be fixed now.

**Comment by kennyp** [comment]
I went ahead and threw in a fix for this in #183 since I was already looking at attributes.
